### PR TITLE
feat(api): Delete Copyright for a File

### LINKED ISSUE #2468: Copyright File-view
File Details (Copyright/email/url/Author, ECC, IPRA) Content, Count, finding, activated/deactivated, delete + User findings with  description, text finding, comment for both activated and deactivated. With all decision options

### PR DESCRIPTION
**Pull request by soham4abc**
## Description

Deletes a copyright for a file using `/file/upload/<uploadId>/item/<ItemId>/copyright/<Hash>`

## Screenshot
Screenshot from 2023-06-17 16-49-34

@GMishx 
@shaheemazmalmmd 

closes #2468

Please consider using the closing keyword if the pull request is proposed to
fix an issue already created in the repository
(https://help.github.com/articles/closing-issues-using-keywords/)

<a href="https://gitpod.io/#https://github.com/fossology/fossology/pull/2478"></a>

### CODE REVIEW

File: src/www/ui/api/documentation/openapi.yaml [hunk 1]
```diff
@@ -2640,6 +2688,21 @@ components:
...
+          description: Status code
+          example: 200
+        content:
+          type: string
+          description: Success message
+          example: "Successfully removed Copyright."
+        type:
+          type: string
+          description: type of message
+          example: "INFO"
```
Comment (GMishx) [inline src/www/ui/api/documentation/openapi.yaml:2705]: Isn't the response object exactly same as `Info` object?

File: src/www/ui/api/Controllers/CopyrightController.php [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+   * Delete copyrights for a particular file
+   *
+   * @param  ServerRequestInterface $request
+   * @param  ResponseHelper         $response
+   * @param  array                  $args
+   * @return ResponseHelper
+   */
+
+
+  public function DeleteFileCopyrights($request, $response, $args)
```
Comment (GMishx) [inline src/www/ui/api/Controllers/CopyrightController.php:85]: Please following coding guidelines.
```suggestion
  public function deleteFileCopyrights($request, $response, $args)
```

**Comment by soham4abc** [comment]
Made the requested changes @GMishx 

Please do let me know if any more changes are required
Thank you!

**Review by GMishx — CHANGES_REQUESTED** [review]

**Review by GMishx — APPROVED** [review]
Changes looks good.
Needs test.

**Review by GMishx — APPROVED** [review]
Tested, working as expected.

**Comment by soham4abc** [comment]
@GMishx should I be removing the test case as of now??

**Comment by GMishx** [comment]
Sure @soham4abc , you can remove test cases. And please rebase the branch for merge.

**Comment by GMishx** [comment]
Closing the branch as merged with 6015f625d917d2f2405cd16086e2516bbb8e8a99

**Pull request closed without merging**